### PR TITLE
added 2023-02-21 Twitter Space

### PR DESCRIPTION
**Pull request by thebarshablog**
Decoding CFPs with Britt Kemp

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4661"></a>

### CODE REVIEW

**Review by maharshi-sinha — APPROVED** [review]
LGTM!

File: data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21",
```
Comment (Panquesito7) [inline data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json:8]: Shouldn't you add the starting hour and finishing hour?

Comment (eddiejaoude) [reply data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json:8]: Yes I think this is important 👍

Comment (thebarshablog) [reply data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json]: @eddiejaoude & @Panquesito7 I made necessary changes.

File: data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21 08:00:00 IST",
+    "end": "2023-02-21 09:30:00 IST"
```
Comment (eddiejaoude) [inline data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json:9]: I don't think that is a valid date/time

Please see the documentation for examples `2022-12-09T16:00:00.000+00:00`
http://localhost:3000/docs/how-to-guides/events

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21 T16:00:00.000+00:00",
+    "end": "2023-02-21 T17:00:00.000+00:00"
```
Comment (kumarsonsoff3) [inline data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json:9]: I think there should be no space between the date & time in this format. 🤔

```suggestion
    "start": "2023-02-21T16:00:00.000+00:00",
    "end": "2023-02-21T17:00:00.000+00:00"
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by thebarshablog** [comment]
It just slipped my mind, how do I access my profile .json file for events etc?

**Pull request merged**